docs/en/observability/profiling-known-issues.asciidoc

```diff
@@ -0,0 +1,22 @@
+[[profiling-known-issues]]
+= Known issues
+
+Universal Profiling has the following known issues:
+
+[discrete]
+== Helm chart `profiling-collector` faulty binary location
+
+_Affected: 8.16.0, 8.16.1, 8.16.2, 8.17.0_
+_Fixed in: 8.16.3+, 8.17.1+_
+
+The installation of Universal Profiling collector using the Helm chart `profiling-collector` produces a Deployment with
+the wrong binary path set in `spec.template.spec.containers['pf-elastic-collector'].command`.
+
+The correct path should be `/home/nonroot/pf-elastic-collector` while the `command` field references the deprecated path
+`/root/pf-elastic-collector`. This issue prevents the collector from starting and as a result, the Deployment has Pods
+in `CrashLoopBackoff` status.
+
+To fix this issue, a manual edit of the Deployment manifest should be performed, removing entirely the `command` field
+from the `spec` section.
+
+This can be performed by either manipulating the Deployment manifest directly or by using the `kubectl edit` command.
```

Comment:
```suggestion
To fix this issue, edit the Deployment manifest to remove the `command` field
from the `spec` section. You can do this by manipulating the Deployment manifest directly or by using the `kubectl edit` command.

Reply:
there's a small correction about the logic, I was probably unclear in the first place.

```suggestion
To fix this issue, a manual edit of the Deployment manifest should be performed, removing entirely the `command` field
from the `spec` section.

This can be performed by either manipulating the Deployment template emitted by Helm or by using the `kubectl edit` command.
```